etc/config/c++.amazon.properties

```diff
@@ -544,6 +544,18 @@ compiler.armv8-full-clang-trunk.options=-target aarch64-linux-gnu --gcc-toolchai
 # An alias of the original name for this compiler needs to be maintained to allow old URLs to continue to work
 compiler.armv8-full-clang-trunk.alias=armv8.5-clang-trunk
 
+################################
+# LLVM-MOS (6502 Clang)
+compiler.mosclang.exe=/opt/compiler-explorer/llvm-mos-trunk/bin/clang
```

Comment:
And as for the compiler id itself, can you call it something like `mosclangtrunk`? It's more resilint to future changes too

Reply:
Done.